packages/react/src/components/Masthead/Masthead.js

```diff
@@ -58,13 +54,46 @@ const Masthead = ({ navigation }) => {
     status.then(result => setStatus(result.user === 'Authenticated'));
   }, []);
 
-  const navigationLinks = navigation.links;
-
   const className = cx({
     [`${prefix}--header__logo`]: true,
```

Comment:
@annawen1 yes good call. this chunk is actually unfinished since `navigation.platform` is still not being used (need to talk to lara/Scott about that)